strtoint/strtoint2.go

```diff
@@ -0,0 +1,12 @@
+package strtoint
+
+import "strconv"
+
+func MyStrToInt2(s string) (int, error) {
+	i, err := strconv.ParseInt(s, 10, 64)
+	if err != nil {
```

Comment:
```suggestion
	return int(i), err
```

Reply:
The same case as prev.

Reply:
https://github.com/VSKut/go-temp/commit/9f9b32d80dfe66ac9f7935afd6bdb715edcc50ae
https://github.com/VSKut/go-temp/commit/b592da9857478c3267a16d23f9e88d6e71d4c0f6